src/Microsoft.DotNet.Files/ContentFiles.cs

```diff
@@ -19,45 +20,68 @@ public ContentFiles(ProjectContext context)
             _context = context;
         }
 
-        public void StructuredCopyTo(string targetDirectory)
+        public void StructuredCopyTo(string targetDirectory, IEnumerable<IncludeEntry> includeEntries)
         {
-            var sourceFiles = _context
-                .ProjectFile
-                .Files
-                .GetContentFiles();
+            if (includeEntries == null)
```

Comment:
@ajaybhargavb it's benefit readability as well as review-ability. in addition, two methods do their own things is better than one method does both things.